bulletin/index.html

```diff
@@ -34,137 +34,136 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
+			</article>
+			<article class="buarticle buarticle-main">
+				 <h3 class="buarticle-main--title">MACILE Summer Academy </h3>
+				<p>The Summer Academy is a rigorous five week program for students in grades 6 to 12. The curriculum includes math, science, engineering, technology, and language. 
+				Courses and activities are designed to stimulate curiosity, creativity, and passion for learning; also to build the students’ skills, analytical abilities, and rational thinking. The program was piloted in July 2007 and 2008.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/SA2007/bbq-2007-012.jpg">
+				<p>The 2007 Summer Academy lasted for three weeks and included mathematics only. 47 students completed the program. They attended from 8:00 AM to 12:00 PM, Monday through Friday, and a healthy breakfast snack was provided. We celebrated the end of the summer with a barbeque.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/SA2007/sa2007-school-9.jpg">
+				<h3 class="buarticle-main--title">People that made it possible in the field</h3>
+				<p>Itabo elementary school director(far left), 6th history and Spanish teacher (far right), and community leaders. They visited the schools, selected students, provided the space, and assisted with logistics</p>
+			</article>
+			<article class="buarticle buarticle-violet">
+				<img src="/images/bulletin/SA2007/group1-sa2007-11.jpg">
+				<p>Summer Academy 2007. Group of 7th-8th grade students from schools in Itabo, Nigua, El 18, Yogo Yogo, and Haina. Far right, Yahaira Pinales, Dominican teacher that recruited the students. </p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Objectives</h3>
+				<ul class="buarticle-features--vision">
+					<li>Nourture the creativity and curiosity of talented young people.</li>
+					<li>Increase acces to challenging and estimulating learning environments.</li>
+					<li>Advance engineering education in K-12</li>
+					<li>Increase learning and understanding.</li>
+				</ul>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Summer Academy 2008</h3>
+				<p>The progam was adjusted in light of the observations made in 2007.  The duration was extended to five weeks and the curriculum was redesigned to include math, engineering technology, Spanish literacy, and activities such as chess, field trips, and guest speakers.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">About the students</h3>
+				<p>Thirty-five students were admitted and completed the program. (MACILE targets talented students in the top 20% of their class.) There was a required orientation and a meeting with the parents .</p>
```

Comment:
Here you spell out "thirty-five". above you used numerals (47 students). The document should remain consistent. Please pick a style format and update.